### PR TITLE
Add unit tests to cover deserialization of Merge layer subtypes

### PR DESCRIPTION
**Pull request by caisq**
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-layers/132)

### CODE REVIEW

File: src/layers/merge_test.ts [hunk 1]
```diff
@@ -376,3 +379,139 @@ describeMathCPUAndGPU('Concatenate Layer: Tensor', () => {
...
+    // ```python
+    // import keras
+    //
+    // input1 = keras.Input(shape=[4])
+    // input2 = keras.Input(shape=[4])
+    //
+    // output = keras.layers.add([input1, input2])
+    // model = keras.Model([input1, input2], output)
+    //
+    // print(model)
```
Comment (bileschi) [inline src/layers/merge_test.ts]: Is this line intentional?  Does it provide value over the call to print(model_json)?

Comment (bileschi) [reply src/layers/merge_test.ts]: same q in concat version below

**Comment by caisq** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[src/layers/merge_test.ts, line 395 at r1](https://beta.reviewable.io/reviews/tensorflow/tfjs-layers/132#-LAKqYlx5XaVCaW3FBME-r1-395:-LAO3VP90h5V1dtzwR-g:b-gawj71) ([raw file](https://github.com/tensorflow/tfjs-layers/blob/9def9784e51bcfc0978555dd489143854b5b9e08/src/layers/merge_test.ts#L395)):*
<details><summary><i>Previously, bileschi (Stanley Bileschi) wrote…</i></summary><blockquote>

same q in concat version below
</blockquote></details>

You are right. These `print(model)` lines are not necessary. Removed.

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/tensorflow/tfjs-layers/132)*

**Comment by nsthorat** [comment]
:lgtm_strong:

---

Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/tensorflow/tfjs-layers/132#-:-LAOAy5D0hXapOz9tlZ-:b-jbsf8z)*

**Pull request merged**